app/(voters)/candidate/[slug]/ElectionTab.tsx

```
@@ -39,8 +39,8 @@ export function ElectionProfileTab({ link }: ElectionProfileTabProps) {
             case BlockType.HEADING:
               const headingClass =
                 block.level === 1
-                  ? `text-2xl font-bold ${color} text-center`
-                  : `text-lg font-semibold ${color}`;
+                  ? `text-4xl font-bold ${color}`
+                  : `text-2xl font-semibold ${color}`;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)